### PR TITLE
Enhance torch.quantization.fuse_modules API to support default module…

### PR DESCRIPTION
**Pull request by gemfield**
This is an enhancement of fuse_modules API.

Currently fuse_modules API need pass the manually specified modules_to_fuse value.
With this enhancement, now we can leave modules_to_fuse as default None value, which will fuse all possible modules in model.

### CODE REVIEW

File: torch/quantization/fuse_modules.py [hunk 1]
```diff
@@ -137,6 +138,22 @@ def fuse_modules(model, modules_to_fuse, inplace=False, fuser_func=fuse_known_mo
     if not inplace:
         model = copy.deepcopy(model)
 
+    if modules_to_fuse is None:
```
Comment (raghuramank100) [inline torch/quantization/fuse_modules.py:141]: Can you add a test case showing how this extension works?
One problem with fusion is that we need a guarantee that the modules are called in sequence. There is no way to automate this unless one uses sequential or explicitly list the modules that are called in sequence.

Comment (gemfield) [reply torch/quantization/fuse_modules.py:141]: Yes, you are right. Without this guarantee from user, we cannot do the automatic fuse in eager mode. So, better drop this PR?

Comment (raghuramank100) [reply torch/quantization/fuse_modules.py:141]: Yes, please. Thanks!

Comment (gemfield) [reply torch/quantization/fuse_modules.py:141]: ok.

**Pull request closed without merging**